pkg/cmd/cmd.go

```diff
@@ -105,6 +106,7 @@ func NewJXCommand(f clients.Factory, in terminal.FileReader, out terminal.FileWr
 	installCommands = append(installCommands, findCommands("cluster", updateCommands)...)
 	installCommands = append(installCommands, findCommands("jenkins token", createCommands, deleteCommands)...)
 	installCommands = append(installCommands, initcmd.NewCmdInit(commonOpts))
+	installCommands = append(installCommands, requirements.NewCmdRequirements(commonOpts))
```

Comment:
Shouldn't this be `jx edit requirements`?

Reply:
I was expecting something a bit more `jx edit userroles` actually.

Reply:
renamed to `jx edit requirements`